Jbrowse-cli broken in release 0.0.1-beta.19

<!-- Please fill out as much of the template as you can. -->

**Describe the bug**
When trying yo use the new version version of Jbrowse-cli, I get errors about the commands not existing.
Example when using the command `jbrowse create ./jbrowse`:
```
    Error: Cannot find module '/path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/lib/commands/create'
    Require stack:
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/plugin.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/config.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/config/lib/index.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/command/lib/command.js
    - /path/to/my/home/.config/yarn/global/node_modules/@oclif/command/lib/index.js
    - /path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/bin/run
    Code: MODULE_NOT_FOUND
```
This error is present with all the cli command like add-track or add-assembly.

I've taken a look in /.config/yarn/global/node_modules/@gmod/jbrowse-cli/ and the folder `lib/` is not present. However in the last version working, 0.0.1-beta.17, it was here. 

**To Reproduce**

Describe here the steps to reproduce the behavior, e.g.:
1. Install the last version of jbrowse-cli: `yarn global add @gmod/jbrowse-cli@0.0.1-beta.19`
2. Try to use jbrowse-cli: `jbrowse create ./test_jbrowse`

**Version:**

jbrowse-cli version 0.0.1-beta.19

**Additional Information**
Directory structure of jbrowse-cli downloaded by yarn
```
tree /path/to/my/home/.config/yarn/global/node_modules/@gmod/jbrowse-cli/
├── bin
│   ├── run
│   └── run.cmd
├── CHANGELOG.md
├── oclif.manifest.json
├── package.json
└── README.md
```

--- a/products/jbrowse-cli/src/commands/create.ts
+++ b/products/jbrowse-cli/src/commands/create.ts
@@ -59,7 +59,6 @@ export default class Create extends JBrowseCommand {
     const { args: runArgs, flags: runFlags } = this.parse(Create)
     const { localPath: argsPath } = runArgs as { localPath: string }
     this.debug(`Want to install path at: ${argsPath}`)
-    console.log('wow')
 
     const { force, url, listVersions, tag } = runFlags
 

--- a/products/jbrowse-cli/tsconfig.json
+++ b/products/jbrowse-cli/tsconfig.json
@@ -7,6 +7,7 @@
     "outDir": "lib",
     "rootDir": "src",
     "strict": true,
+    "noEmit": false,
     "target": "es2017"
   },
   "include": ["src"],

